apps/cookbook/src/app/examples/slides-example/slides-example.shared.scss

```diff
@@ -8,3 +8,8 @@
   --padding-start: #{utils.size('s')};
   --padding-end: #{utils.size('s')};
 }
+
+.stretch-vertically {
+  height: 100%;
+  justify-content: start;
+}
```

Comment:
I think we should figure out a way to make this an 'official' API instead of some custom styling added per component in projects. 

The only flexible way I can think of, aside from adding this to global styles, is to create an additional directive that sets these styles on its host element. This allows consumers to use kirby-card or any other element for individual slides:


```
import { Directive, ElementRef, Renderer2 } from '@angular/core';

@Directive({
  // eslint-disable-next-line @angular-eslint/directive-selector
  selector: '[slideStretch]',
})
export class SlideStretchDirective {
  constructor(private elementRef: ElementRef, private renderer: Renderer2) {
    this.renderer.setStyle(this.elementRef.nativeElement, 'height', '100%');
    this.renderer.setStyle(this.elementRef.nativeElement, 'justify-content', 'start');
  }
}

```

Then consumers will just have to do something like this:
```
<kirby-slides [slides]="slides" [title]="'Title'" [showNavigation]="true">
  <kirby-card *kirbySlide="let slide; let i = index" slideStretch [hasPadding]="true">
    <kirby-card-header [title]="slide.title" [subtitle]="slide.subtitle"></kirby-card-header>
    <div class="card-content">
      {{ slide.cardContent }}
    </div>
  </kirby-card>
</kirby-slides>
```

Reply:
Stretching the content of the slider vertically or height is in my opinion entirely up to the implementor of the designsystem. The implementation with properties `height` and `justify-content` is not always applicable, but needed in the cookbook example. It's not necessary in our implementation.